### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix: https://github.com/PaddlePaddle/Paddle/issues/7263

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+Distributions that PaddlePaddle supports officially includes:
+
+- CentOS 6 and above
+- Ubuntu 14.04 and above
+- Other distributions that similar to the above two versions, e.g. SUSE or Debian.
 
 Borrowed scripts from https://github.com/numenta/manylinux to build manylinux build environments
 based on different versions of CUDA and CUDNN base images, so that we can build PaddlePaddle
-using these images to release different versions of PaddlePaddle binaries.
+using these images to release different CPU/GPU versions of PaddlePaddle binaries.
```
Comment (Yancey1989) [inline README.md]: The docker images from **buildtools** also support different Python ABI with the specified ENV `PYTHON_ABI`.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,20 @@
 # buildtools
-Contain dockerfiles for building paddle on different distros, cuda versions etc.
+
```
Comment (wangkuiyi) [inline README.md]: Thanks for the rewriting of the first paragraph. But still it doesn't explain the motivation of manylinux support. Indeed, as old PaddlePaddle runs as an enhanced Python interpreter, it is not supposed to be released as a built Python module and thus doesn't have to follow the manylinux standard. Only Fluid makes it sense to follow the standard.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,20 @@
...
+Buildtools is a set of Docker images for build PaddlePaddle release binaries.
+These Docker images supports [manulinux1](https://www.python.org/dev/peps/pep-0513/)
+so that we can guarantee that our `whl` packages can be installed and run on most
+linux distributions, including some "old" distributions like CentOS6.
+
+Distributions that PaddlePaddle supports officially includes:
```
Comment (wangkuiyi) [inline README.md]: Again, a document need to explain why we support the following distros.

File: README.md [hunk 4]
```diff
@@ -1,9 +1,20 @@
...
+
+- CentOS 6 and above
+- Ubuntu 14.04 and above
+- Other distributions that similar to the above two versions, e.g. SUSE or Debian.
```
Comment (wangkuiyi) [inline README.md]: This is very inaccurate. How to define "similar"?

File: README.md [hunk 5]
```diff
@@ -1,9 +1,20 @@
...
 
 Borrowed scripts from https://github.com/numenta/manylinux to build manylinux build environments
 based on different versions of CUDA and CUDNN base images, so that we can build PaddlePaddle
-using these images to release different versions of PaddlePaddle binaries.
+using these images to release different CPU/GPU or python [ABI](https://www.python.org/dev/peps/pep-0513/)
```
Comment (wangkuiyi) [inline README.md]: What does it mean "Python version of PaddlePaddle"? Do you mean the source code under the `python` sub-directory?

File: README.md [hunk 6]
```diff
@@ -1,9 +1,20 @@
...
+versions of PaddlePaddle binaries.
 
 **IMPORTANT** Since this version of manylinux1_x86_64 is based on centos-6 (instead of centos-5), it is technically non-compliant with manylinux1 policy of https://www.python.org/dev/peps/pep-0513/, which requires centos-5 system libraries. However, as suggested in discussions https://mail.python.org/pipermail/wheel-builders/2016-July/000175.html, the wheels produced with the centos-6-based image should still be compatible with many distros (except centos-5), and a future manylinux2 policy may indeed be based on centos-6 as well.
```
Comment (wangkuiyi) [inline README.md]: centos-6 ==> CentOS 6 ?

File: README.md (hunk 2 above)
Comment (typhoonzero) [reply README.md]: PEP 513 is a standard that restricting built external binaries compatibility in the `whl` package: https://www.python.org/dev/peps/pep-0513/#key-causes-of-inter-linux-binary-incompatibility. Both fluid and legacy PaddlePaddle have to pack those C++ written binaries (`core.so` and `_swig_paddle.so`).

File: README.md (hunk 3 above)
Comment (wangkuiyi) [reply README.md]: manylinux suggests building on CentOS 5. However, the minimum version of CentOS supports CUDA is 6.

File: README.md (hunk 5 above)
Comment (wangkuiyi) [reply README.md]: https://www.python.org/dev/peps/pep-0513/#ucs-2-vs-ucs-4-builds

Comment (wangkuiyi) [reply README.md]: https://pypi.python.org/pypi/tensorflow-gpu/1.5.0rc0

**Pull request merged**